### PR TITLE
Gitpod

### PR DESCRIPTION
**Pull request by Paalar**
<a href="https://gitpod.io/#https://github.com/appKom/scibble/pull/37"></a>

### CODE REVIEW

**Comment by Jorrre** [comment]
Very good commit message convention, clear and concise

File: .gitpod.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,57 @@
+FROM gitpod/workspace-full
+
+# FLutter
```
Comment (Jorrre) [inline .gitpod.Dockerfile]: Small typo

**Comment by Jorrre** [comment]
Tried opening gitpod and got this error
```
Android sdkmanager tool not found (/usr/lib/android-sdk/tools/bin/sdkmanager).
Try re-installing or updating your Android SDK,
visit https://flutter.dev/docs/get-started/install/linux#android-setup for
detailed instructions.
```
Should this be fixed, or was this what you meant by android emu not supported?

**Comment by Paalar** [comment]
I can try to look into the error, but I think it will occur anyway because of ```flutter upgrade```

**Comment by Paalar** [comment]
I haved fixed the issues @Jorrre  ! Should be working fine now.

**Review by Jorrre — APPROVED** [review]

**Pull request merged**